app/app.go

```diff
@@ -173,6 +178,12 @@ func NewApp(
 	logger log.Logger, db dbm.DB, tio io.Writer, loadLatest bool, invCheckPeriod uint, skipUpgradeHeights map[int64]bool,
 	homePath string, appOpts servertypes.AppOptions, options ...func(*bam.BaseApp),
 ) *AkashApp {
+	// find out the genesis time, to be used later in inflation calculation
+	if genDoc, err := tmtypes.GenesisDocFromFile(filepath.Join(homePath, "config/genesis.json")); err != nil {
+		panic(err)
```

Comment:
This is panicking in the test environment, but not locally. Will have to debug it further.

Reply:
Fixed